rust/linux-bootloader/src/efivars.rs

```diff
@@ -19,6 +19,11 @@ use uefi::{
 
 use bitflags::bitflags;
 
+/// Fetch the PARTUUID of a given disk
+/// FIXME(security): UEFI makes no practical guarantee about the unicity of PARTUUID
```

Comment:
Is this really a FIXME? This sounds like a discussion about security nuances. This should be in a separate document.

Reply:
it's related to https://github.com/systemd/systemd/issues/28491

Reply:
but yes